src/components/NavigationAccount.vue

```diff
@@ -43,6 +43,12 @@
 			<ActionInput icon="icon-add" @submit="createFolder">
 				{{ t('mail', 'Add folder') }}
 			</ActionInput>
+			<ActionButton v-if="!isFirst" icon="icon-triangle-n" @click="changeAccountOrderUp">
+				{{ t('mail', 'Move Up') }}
```

Comment:
We always only capitalize the first word. :) So "Move up". Strangely enough, "Move down" is correct. ;)

Reply:
Send a PR? :)

Reply:
good catch,thats definitely a typo, my typo, sorry.

Reply:
@GretaD no worries – could you fix it? :)